src/main/java/net/william278/velocitab/Velocitab.java

```diff
@@ -132,10 +132,6 @@ private void loadHooks() {
 
     private void prepareScoreboardManager() {
         if (settings.isSortPlayers()) {
-            if (!Hook.isPluginAvailable(this, "protocolize")) {
```

Comment:
I had to make `Hook#isPluginAvailable` public when I added this ugly check; it should probably now be made private again.